standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParserSuites/TypeParserFromLongTypeSuite.scala

```diff
@@ -45,8 +46,21 @@ class TypeParserFromLongTypeSuite extends TypeParserSuiteTemplate {
     }
   }
 
-  test("Within the column - type stays") {
-    doTestWithinColumn(input)
+  override protected def createErrorCondition(srcField: String, target: StructField, castS: String): String = {
```

Comment:
Why you are overriding this method in several test suites with identical implementation?